src/java.base/share/classes/sun/net/www/protocol/http/AuthenticationInfo.java

```diff
@@ -218,26 +213,20 @@ private static void requestCompleted (String key) {
     /** The shortest path from the URL we authenticated against. */
     String path;
 
-    /**
-     * A key identifying the authenticator from which the credentials
-     * were obtained.
-     * {@link AuthenticatorKeys#DEFAULT} identifies the {@linkplain
-     * java.net.Authenticator#setDefault(java.net.Authenticator) default}
-     * authenticator.
-     */
-     String authenticatorKey;
+    /** A cache to obtain credentials from if the default cache not to be used */
+    transient AuthCacheImpl authcache; // may be null
```

Comment:
Now that `AuthCacheValue` is no longer `Serializable` in this PR (and thus this `AuthenticationInfo` too isn't serializable), is the use of `transient` here and on some other fields in this class, needed anymore?

Reply:
Good point.